scalafix-core/shared/src/main/scala/scalafix/internal/rule/OptionConstructor.scala

```diff
@@ -0,0 +1,23 @@
+package scalafix.internal.rule
+
+import scala.meta._
+
+import scalafix.{Patch, SemanticRule, SemanticdbIndex}
+import scalafix.rule.{RuleCtx, RuleName}
+
+final case class OptionConstructor(index: SemanticdbIndex)
```

Comment:
Minor, but I think something like `NoSomeConstructor` or `PreferOptionConstructor` would be more inline with other rules names.

/cc @olafurpg

Reply:
This will make it much more consistant with the `DisableSyntaxConfig.noSomeConstructor`